File: app/Models/Comentario.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Comentario extends Model
{
    use HasFactory;

    protected $table = 'comentarios';

    protected $fillable =
        [
            'id',
            'id_cliente',
            'descripcion',
            'created_at',
            'updated_at'
        ];
}

File: models/comentario.go
package models

import "time"

type Comentario struct {
	ID          uint      `json:"id" gorm:"primaryKey"`
	Descripcion string    `json:"descripcion"`
	IDCliente   uint      `json:"id_cliente"`
	IDReceta    uint      `json:"id_receta"`
	CreatedAt   time.Time `json:"created_at"`
	UpdatedAt   time.Time `json:"updated_at"`
}

func (Comentario) TableName() string {
	return "comentarios"
}
